photon-api/src/main/scala/com/linkedin/photon/ml/function/svm/DistributedSmoothedHingeLossFunction.scala

```diff
@@ -113,14 +113,16 @@ object DistributedSmoothedHingeLossFunction {
    */
   def apply(
       configuration: GLMOptimizationConfiguration,
-      treeAggregateDepth: Int): DistributedSmoothedHingeLossFunction = {
+      treeAggregateDepth: Int,
+      index: Int = -1): DistributedSmoothedHingeLossFunction = {
```

Comment:
Same.

Reply:
In addition, parameter name and type should change (see below):
```
interceptIndexOpt: Option[Int] = None
```